app/dropshop/products/serializers.py

```diff
@@ -32,7 +32,7 @@ def _get_price_data(self, instance):
                 'currency': None,
                 'percent': None,
                 'price_difference': None,
-                'error': True,
+                'error': False,
```

Comment:
@AndarooLee What's this for?

Reply:
So that front end shows loading instead of error